### PR TITLE
JSON-RPC: Update for --jsonrpcbindip

### PR DESCRIPTION
**Pull request by Rob-NY**
**Short description of changes**

Documentation for --jsonrpcbindip 

**Context: Fixes an issue? Related issues**

Relates to PR #2917 in jamulus core.

**Status of this Pull Request**

**What is missing until this pull request can be merged?**

Translations

**Does this need translation?**

Probably, not sure.

## Checklist

- [X] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [X] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [X] I'm sure that this Pull Request goes to the correct branch

### CODE REVIEW

**Review by gilgongo — APPROVED** [review]
Is this feature in for the next release? If so, any objections to merging it?

File: wiki/en/Include-Shared-Commands.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 - `-p` or `--port`           Sets the local UDP port number. Default is 22124
 - `--jsonrpcport`            Enables JSON-RPC API server to control the app, set TCP port number (EXPERIMENTAL, APIs might change; only accessible from localhost). Please see [the JSON-RPC API Documentation file](https://github.com/jamulussoftware/jamulus/blob/master/docs/JSON-RPC.md).
 - `--jsonrpcsecretfile`      Required when using `--jsonrpcport`. Sets a path to a text file containing an authentication string for getting access to the JSON-RPC API.
+- `--jsonrpcbindip` 		 The IP address the JSON-RPC server should bind to. (optional, defaults to 127.0.0.1)
```
Comment (hoffie) [inline wiki/en/Include-Shared-Commands.md:10]: The spacing looks off at least here on Github, probably as there's a tab while other places use spaces.

**Review by hoffie — COMMENTED** [review]
This feature was merged in https://github.com/jamulussoftware/jamulus/pull/2917.

File: wiki/en/Include-Shared-Commands.md (hunk 1 above)
Comment (mcfnord) [reply wiki/en/Include-Shared-Commands.md:10]: None of this will render nicely. Seems like a markdown table is needed.

Comment (gilgongo) [reply wiki/en/Include-Shared-Commands.md:10]: We used to have that and it made it even harder to read :-)

Comment (mcfnord) [reply wiki/en/Include-Shared-Commands.md:10]: https://tucc.us/u/ouchy.html

Maybe we could roll our own HTML rather than using Markdown's design.

Comment (mcfnord) [reply wiki/en/Include-Shared-Commands.md:10]: [Is this harder to read?](https://github.com/jamulussoftware/jamuluswebsite/pull/877#issuecomment-1336112230)

Comment (gilgongo) [reply wiki/en/Include-Shared-Commands.md:10]: Not sure if it's wise to use in-line HTML (if that's what's happening - or it is working with the CSS?). Also it needs to work with https://jamulus.io/wiki/Software-Manual#command-line-options - and the Server manual - that's why it's an include.

Comment (mcfnord) [reply wiki/en/Include-Shared-Commands.md:10]: Yes, this is in-line HTML with a pinch of CSS. I am curious what Markdown would make, and haven't looked yet. Markdown lets HTML co-exist inline (user beware, sure), and presumably we have our own CSS such that we can tweak the standard table if need be.

Comment (gilgongo) [reply wiki/en/Include-Shared-Commands.md:10]: Actually just tried it and the markdown doesn't look too bad (I think before we had more text in the cells so the rows looked odd or something). Just means the Client manual will have two tables but that doesn't look awful. It's easier to read. I'll give it a go.

Comment (gilgongo) [reply wiki/en/Include-Shared-Commands.md:10]: I'll wait for the PR to finalise before making the change though. Making a note on my PR so I don't forget.

**Review by ann0see — APPROVED** [review]

**Comment by ann0see** [comment]
Thank you very much!

**Pull request merged**